### ISSUE TITLE
[Feature Request] Support existing MicroG install

### ISSUE THREAD

**Issue by Captn138** [revancedapps/revancedmanager#10]
Hey there, I already have MicroG on my phone, but YTRevanced insists on installing its own MicroG. If there is no technical problem, would it be possible to check for an already existing MicroG instance and registering with that one ?

**Comment by revancedapps** [comment]
Yes, it's possible. Basically, the ReX's build signature differs from the Revanced team's. All you need to do is check the signature before installing. I will support this feature in the upcoming build.

**Comment by Captn138** [comment]
That's actually very nice.
Will there be a way to input the name of the MicroG instance to use ?
Because there are so mnay forks of the project.
For example, I'm on /e/os and the MicroG instance has the com.google.android.gms name...
